classes/Dispatcher.php

```diff
@@ -681,7 +740,8 @@ public function validateRoute($route_id, $rule, &$errors = array())
      * @param bool $use_routes If false, don't use to create this url
      * @param string $anchor Optional anchor to add at the end of this url
      */
-    public function createUrl($route_id, $id_lang = null, array $params = array(), $force_routes = false, $anchor = '', $id_shop = null)
+    public function createUrl($route_id, $id_lang = null, array $params = array(), $force_routes = false, $anchor = '',
```

Comment:
Care about IED's autoformat.
Sometimes it doesn't respect PSR (for instance, here there is a problem with [multi-line method arguments](http://www.php-fig.org/psr/psr-2/#44-method-arguments))